students/grzegorz_bajorski/lesson_03_functions/the_collatz_seq.py

```diff
@@ -0,0 +1,17 @@
+def collatz(number):
```

Comment:
Please have a look at comments to input_validation.py and fix them here as well.